test/unit/Cover/setup.js

```diff
@@ -138,13 +140,13 @@ async function setup () {
   }
 
   // add products
-  await cover.connect(accounts.advisoryBoardMembers[0]).addProduct({
+  await cover.connect(accounts.advisoryBoardMembers[0]).addProducts([{
     productType: '1',
     productAddress: '0x0000000000000000000000000000000000000000',
     coverAssets: '1', // ETH supported
```

Comment:
CoverMockPool is incorrectly initialized. You want ETH here but the assets array only contains DAI (index 0). So the value of coverAssets of 1 is actually DAI instead of ETH. Make sure you initialize the asset on index 0 to ETH just like in Pool.sol before adding other assets.

Reply:
fixed